research/conditional/moe_layers/token_choice.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         init_scale: float,
         routing_top_k: int = 1,
         use_einsum: bool = False,
+        vectorize: bool = False,
     ):
```

Comment:
Add assert / exception somewhere if vectorize and routing_top_k > 1 don't work together.